addons/mrp/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'Manufacturing',
-    'version': '2.0',
+    'version': '2.1',
```

Comment:
No need for this, we don't increment the version of the manifest for every PR made in it. :slightly_smiling_face:

Reply:
This is to trigger migration script to fix existing data when upgrading module mrp

Reply:
@clesgow Please let me know why?